schema/omega-services.async.api.yaml

```diff
@@ -777,6 +837,195 @@ components:
       type: boolean
       description: true to indicate that this Agent is also considered an Authority File, false otherwise
 
+    RecordSummary:
+      type: object
+      required:
+        - identifier
+        - type
+        - creator
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecordDescriptionSummary'
+
+    RecordFull:
+      type: object
+      required:
+        - identifier
+        - type
+        - creator
+        - current-description
+        - description
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        type:
+          $ref: '#/components/schemas/RecordType'
+        creator:
+          type: array
+          items:
+            $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the Creating Body of the Record concept, i.e. a URI to an Agent
+        current-description:
+          $ref: '#/components/schemas/GeneralIdentifier'
+          description: the identifier of the current description of the Record concept
+        description:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecordDescriptionFull'
+
+    RecordDescriptionSummary:
+      type: object
+      required:
+        - identifier
+        - label
+        - abstract
+        - access-rights
+        - is-part-of
+        - version-timestamp
+      properties:
+        identifier:
+          $ref: '#/components/schemas/GeneralIdentifier'
+        label:
```

Comment:
I am assuming that for a record the label would be the PAC-ID - but what about the CCR? I don't see anywhere to for that to go and the UI is currently expecting both the PAC-ID and CCR.

Reply:
For a record the PAC-ID will be the `identifier` field. The CCR will go in the `secondary-identifier`.

Reply:
OK. In that case the `secondary-identifier` should probably go into the `RecordDescriptionSummary` rather than the `RecordDescriptionProperties` as this is used in the [View Work Set page](https://ctd-omega-editorial-prototype.herokuapp.com/iteration-3/view-work-set) - as are the covering dates so they should probably also be in the summary.

I'm still not clear what would go into the `label` field though - and this is marked as _required_.

Reply:
So the `label` is a human friendly "title" / "description" / "summary", or similar. I felt that it was a useful thing to have for display purposes. For a record it could be the first *n* words of the Scope and Content perhaps?

> In that case the secondary-identifier should probably go into the RecordDescriptionSummary rather than the RecordDescriptionProperties as this is used in the [View Work Set page](https://ctd-omega-editorial-prototype.herokuapp.com/iteration-3/view-work-set)

I have never really liked having the CCR in the Editorial Application, it has too many problems. However, as that is needed for the time being I have added it to the summary as requested.

Reply:
As far as the label is concerned, we need to know what the users want to see.

Reply:
Also the CCR doesn't fit the model of the `GeneralTypedIdentifier` which applies to `secondary-identifier`. The CCR is just a string value for the property `cat:classicCatalogueReference`.

Reply:
> Wouldn't the scope and content go in the abstract? In which case the label would just be duplicating what is in there.

@rwalpole I had envisaged the `label` as being much shorter than the `abstract`.

> Also the CCR doesn't fit the model of the GeneralTypedIdentifier which applies to secondary-identifier

@rwalpole Sorry, I am not sure what you mean by `doesn't fit the model`. It would seem to me that the following or similar would be valid according to the AsyncAPI schema:
```json
"secondary-identifier": [
  {
    "identifier": "WO 95/10/1",
    "type": "http://catalogue.nationalarchives.gov.uk/classicCatalogueReference"
  }
]
```

Reply:
I see what you are saying and we can certainly do that. I think my question arises because there seems to be a mismatch within the model. We currently have the CCR like so:
```
cat:COAL.2022.N373.P.2 cat:classicCatalogueReference  "COAL 80/2052/9" .
```
Whereas there are other idenifiers such as:
```
cat:COAL.2022.N373.P.2 schema:identifier [
    a schema:identifier ;
    schema:propertyID  <http://www.nationalarchives.gov.uk/ont.iaid> ;
    schema:value       "C12085388"
] .
```
Which makes me think the CCR should be more like:
cat:COAL.2022.N373.P.2 schema:identifier [
    a schema:identifier ;
    schema:propertyID  cat:classicCatalogueReference;
    schema:value       "COAL 80/2052/9"
] .
```

Reply:
@adamretter yes we can absolutely do that. I was confused why we had two different ways of modelling alternative identifiers in the OCDM but I re-read [your blog post](https://blog.adamretter.org.uk/vocabulary-reuse-part2/) and I think I understand now.